source/tab/LoginTracker.js

```diff
@@ -0,0 +1,27 @@
+import { getCurrentTitle, getCurrentURL } from "./page.js";
+
+let __sharedTracker = null;
+
+export default class LoginTracker {
+    constructor() {
+        this.username = "";
```

Comment:
you can just class properties here (similar to `state = ...`) etc.

Reply:
👍